mobile/lib/modules/backup/services/backup.service.dart

```diff
@@ -147,7 +134,8 @@ class BackupService {
     return result;
   }
 
-  Future<List<AssetEntity>> _removeAlreadyUploadedAssets(
+  /// Returns a new list of assets not yet uploaded
```

Comment:
Let's ask some comments for this method that it is currently being used only for background service.